priv/python3/erlport/erlang.py

```diff
@@ -127,8 +127,8 @@ def set_message_handler(self, handler):
         self.handler = handler
 
     def _check_handler(self, handler):
-        # getargspec will raise TypeError if handler is not a function
-        args, varargs, _keywords, defaults = getargspec(handler)
+        # getfullargspec will raise TypeError if handler is not a function
+        args, varargs, _keywords, defaults = getfullargspec(handler)
```

Comment:
You can't do this just replacing func name, [`inspect.getfullargspec`](https://docs.python.org/3.10/library/inspect.html#inspect.getfullargspec) returns tuple of 7 elements, not 4
Valid fix:
```python
args, varargs, _keywords, defaults, _kwonlyargs, _kwonlydefaults, _annotations = getfullargspec(handler)
# or
args, varargs, _keywords, defaults, *_rest = getfullargspec(handler)
```

Reply:
Fixed, thanks. :see_no_evil: